python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_estimator.py

```diff
@@ -605,6 +621,9 @@ def load(self, model_path):
                 self.state_dict = [re.state_dict() for re in res]
         else:
             self.state_dict = res.state_dict()
+        if self.model_creator is None:
+            self.model_creator = lambda config: res
```

Comment:
I think it is better to put torch.load again in the creator instead of directly pass res
If model_creator is None, we can assume user load the entire model right? Maybe also add a check?

Reply:
That cannot be implemented since we load the entire model instead of model state_dict, there will be a serialization error when loading the entire model on spark executor.

Reply:
Why load the entire model on each runner would have serialization error? no serialization is involved...

Reply:
torch needs to serialize the entire model to get the model class.